### ISSUE TITLE
Feature request: Pause/resume transfers

### ISSUE THREAD

**Issue by kierenj** [Microsoft/AzureStorageExplorer#1279]
I'm archiving fairly large amounts of data, but our upload bandwidth is limited.  It would be incredibly useful to be able to pause transfers and resume them later.

**Comment by frode** [comment]
We have been transferring large virtual machine images lately and when a upload files at somewhere between 90-99% - you need to restart uploading 200GB + large files gets quite boring and ineffective use of time. Please make this happen ;-)

**Comment by cdeil** [comment]
I had a large download from ADLS gen2 to my local Macbook that failed half-way because of flaky network connectivity (for a minute, after 2 hours of download).

Can you add a "resume" or "continue" download feature for such cases?

Would it be better if I filed a separate feature request, or is this comment here OK?

**Comment by rj-platt** [comment]
Yes, a job history with pause/resume/auto-resume is really needed. I was transferring several TB when the host server I was running Storage Explorer on was rebooted for maintenance. Luckily, now that Storage Explorer uses AZCopy, I was able to resume it from command line using AZCopy on that server by changing to:
cd "C:\Program Files (x86)\Microsoft Azure Storage Explorer\resources\app\node_modules\se-az-copy-exe-win\dist\bin"

Login:
azcopy_windows_amd64.exe login --tenant-id=xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxx

Get list of jobs:
azcopy_windows_amd64.exe jobs list

show status of the job:
azcopy_windows_amd64.exe jobs show 00000000-1111-2222-3333-4444444444
Percent Complete (approx): 66.4
Final Job Status: InProgress

Resume the job:
azcopy_windows_amd64.exe jobs resume 00000000-1111-2222-3333-4444444444

**Comment by Ksingh108** [comment]
I could use this feature too, for same reasons others have mentioned.

**Comment by zottinor** [comment]
I`m looking someway to pause and resume as well

**Comment by xuyuji9000** [comment]
May I ask how is development of this feature going?

My non-dev colleague was asking me about this pause and resume feature of `Microsoft Azure Storage Explorer`.

Much appreciated if there is some feedback on this.

**Comment by jinglouMSFT** [comment]
Thank you all for the requests. Once our dependencies support pause and resume, we will add it in Storage Explorer. We know that this is one of the highly asked features. We will prioritize it in our backlog.  Thanks again!

**Comment by ElvisVangDBA** [comment]
Please ensure this feature request includes resuming a broken download where it left off. We have large files that never completes downloading due to various reasons, and have to start over and attempt multiple times.

**Comment by kylejohnstonGV** [comment]
I'm surprised this feature hasn't been added yet!

**Comment by greg-bellasis** [comment]
Come on man!  This is sorely needed or at least a documented work around similar to something posted above. 
 https://github.com/microsoft/AzureStorageExplorer/issues/1279#issuecomment-632120246
